core/api/src/models/Group.ts

```diff
@@ -62,6 +63,9 @@ const STATE_TRANSITIONS = [
   { from: "ready", to: "deleted", checks: [] },
 ];
 
+@DefaultScope(() => ({
+  where: { state: "ready" },
```

Comment:
I think this will drop "initializing" and "updating" --- that's probably not the intent.

Reply:
Yep- once the tests are green, I'm going to investigate compound or ~= scopes for this one